### PR TITLE
Fixed table in View Events

### PR DESCRIPTION
**Pull request by lfallert**
This is a fix on the table that displays when searching, so that extra table data does not appear on the events list. Test by making sure that when viewed searched events, that the table headers match the table rows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +30,7 @@
     "@angular/cli": "~8.2.2",
     "@angular/compiler-cli": "~8.2.0",
     "@angular/language-service": "~8.2.0",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
```
Comment (CNichole726) [inline package.json:34]: also this seemed to have been missing so I added it back... I hope it is there for the next reviewer! If not, let me know so I can do some more troubleshooting with it

**Review by CNichole726 — APPROVED** [review]
Working as intended, thanks!

File: package.json (hunk 1 above)
Comment (aeydie) [reply package.json:34]: It was missing for me too - but I think Laura has it fixed up now. :)

**Review by aeydie — APPROVED** [review]
Looks good - Laura will fix the import NPM weird stuff, but otherwise looks as it should! :)

**Pull request merged**